scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/job/InternalJobFactory.java

```diff
@@ -143,14 +144,26 @@ private static InternalJob createJob(TaskFlowJob userJob) throws JobCreationExce
         // keep an initial job content
         job.setTaskFlowJob(userJob);
         Map<Task, InternalTask> tasksList = new LinkedHashMap<>();
-        boolean hasPreciousResult = false;
+
+        ExecutorService pool = Executors.newWorkStealingPool(8);
 
         for (Task t : userJob.getTasks()) {
-            tasksList.put(t, createTask(userJob, job, t));
+            pool.execute(new Runnable() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)